Makefile

```diff
@@ -1,7 +1,12 @@
 # Image URL to use all building/pushing image targets
-IMG ?= <repository>
+IMG ?= cnwan/cnwan-operator:v0.2.1
```

Comment:
We are referencing the image here in the Makefile and in one of the YAML files. We should find a way to have it in a single place. Maybe a single place for the current version  number, and we reference just the version variable everywhere?

Reply:
There are now scripts that can be used to automate deployment/removal, leaving `make` to only advanced users.
They are under `/scripts`. Let me know what you think! :)

Reply:
UPDATE: I have also included `get-latest.sh` which pulls the latest container version from DockerHub. So the makefile and the scripts use that one and we don't have to manually specify it every time.

Reply:
This is awesome, exactly what I had in mind. Just two comments on the changes:
- Please change `<repository>` to all caps so that it's clear that it has to be replaced
- I think it would be better to move `05_deployment.yaml` to `05_deployment.yaml.tpl` or similar to denote that it's a template that needs some replacement text and modify the scripts to generate the non-tpl version, which should be added to `.gitignore` That way, when you run the install scripts, you don't have a modified file from the Git perspective, the repository remains "clean".

Reply:
I changed `<repository>` to `{CONTAINER_IMAGE}` and gitignored `*_generated.*` :)

Reply:
Excellent, thank you, looks great now!